### PR TITLE
removed acs from transcription

### PR DESCRIPTION
**Pull request by qdap-bp**
This pull request primarily focuses on improving the chat functionality in the application. The changes mainly revolve around the components and services related to chat, enhancing the user interface, and improving the handling of chat files and messages. 

User Interface Improvements:
* [`src/components/chat/chat-file-transcription.tsx`](diffhunk://#diff-1999aca0416f0b7591be8cb7d654a40e1f1fc14427393fbef6ea7a82be1f9ec9L51-R51): Updated the CSS class in the `ChatFileTranscription` component to use a container class for better alignment.
* [`src/features/chat/chat-menu/menu-items.tsx`](diffhunk://#diff-af88df112ea4a4ecf9561ba98adc52dc8e77e941714779bba238c9481d764152L49-R49): Increased the opacity of the modal background for better visibility.
* [`src/features/chat/chat-ui/chat-file/chat-file-ui.tsx`](diffhunk://#diff-23dc59e35f50a335f6266f85434debf2eb4b0ab1cc70a842ce59bc67d6e5ab94L27-R27): Removed the restriction of only supporting `.wav` files for audio and now supports all audio and video formats.
* [`src/features/chat/chat-ui/chat-message-container.tsx`](diffhunk://#diff-66d171166e4023058f819822019dea9362992c85c7c5e769a1c207017002db41L25-R27): Modified the default selected tab in the `ChatMessageContainer` component based on the availability of messages, and updated the tab selection when loading. The component's layout was also updated to better handle chat messages and transcriptions. [[1]](diffhunk://#diff-66d171166e4023058f819822019dea9362992c85c7c5e769a1c207017002db41L25-R27) [[2]](diffhunk://#diff-66d171166e4023058f819822019dea9362992c85c7c5e769a1c207017002db41R43) [[3]](diffhunk://#diff-66d171166e4023058f819822019dea9362992c85c7c5e769a1c207017002db41L66-R70) [[4]](diffhunk://#diff-66d171166e4023058f819822019dea9362992c85c7c5e769a1c207017002db41L79-L90) [[5]](diffhunk://#diff-66d171166e4023058f819822019dea9362992c85c7c5e769a1c207017002db41L103-R102)
* [`src/features/chat/chat-ui/chat-ui.tsx`](diffhunk://#diff-3421233431c80dad6d74c9f867a0e751f32c466139bd2603d83bccae063013dfL13-R21): Updated the `ChatUI` component to render the `ChatMessageContainer` if there are either messages or documents available.

Chat File and Message Handling:
* [`src/features/chat/chat-services/chat-thread-service.ts`](diffhunk://#diff-bddad34450c7e7af3d06ffd22ebcde980e85c8af1017de49bfc10f96c7084758L355-R359): Enhanced the `FindChatThreadByTitleAndEmpty` function to also check for documents in addition to messages when determining if a chat thread is empty.
* [`src/features/chat/chat-ui/chat-file/use-file-selection.ts`](diffhunk://#diff-197f80022f00ebd5da450a4a63b59aa9cf193ffd2b87758a60c0c74f48a5792bR27-R36): Updated the `useFileSelection` function to handle the uploading and indexing of files more efficiently. The function now sets a more descriptive upload button label based on the chat type, simplifies the handling of file names, and improves error handling. [[1]](diffhunk://#diff-197f80022f00ebd5da450a4a63b59aa9cf193ffd2b87758a60c0c74f48a5792bR27-R36) [[2]](diffhunk://#diff-197f80022f00ebd5da450a4a63b59aa9cf193ffd2b87758a60c0c74f48a5792bL51-R53) [[3]](diffhunk://#diff-197f80022f00ebd5da450a4a63b59aa9cf193ffd2b87758a60c0c74f48a5792bL72)

### CODE REVIEW

File: src/features/chat/chat-ui/chat-context.tsx [hunk 1]
```diff
@@ -92,11 +80,7 @@ export const ChatProvider: FC<Prop> = props => {
     id: props.id,
     body: chatBody,
     initialMessages: props.chats,
-    onFinish: (lastMessage: Message) => {
```
Comment (kpqdap) [inline src/features/chat/chat-ui/chat-context.tsx:95]: Is this not required?

File: src/features/chat/chat-ui/chat-file/use-file-selection.ts [hunk 2]
```diff
@@ -24,51 +24,44 @@ export const useFileSelection = (
   const onFileChange = async (formData: FormData): Promise<void> => {
     try {
       setIsUploadingFile(true)
-      setUploadButtonLabel("Uploading file...")
+      if (chatBody.chatType === "audio") {
+        setUploadButtonLabel("Uploading and transcribing audio...")
```
Comment (kpqdap) [inline src/features/chat/chat-ui/chat-file/use-file-selection.ts:28]: audio or.... Video?

File: src/features/chat/chat-ui/chat-context.tsx (hunk 1 above)
Comment (qdap-bp) [reply src/features/chat/chat-ui/chat-context.tsx:95]: removed

File: src/features/chat/chat-ui/chat-file/use-file-selection.ts (hunk 2 above)
Comment (qdap-bp) [reply src/features/chat/chat-ui/chat-file/use-file-selection.ts]: updated to "file"

**Review by kpqdap — APPROVED** [review]

**Pull request merged**